### DISCUSSION TITLE
.NET Foundation - Should we be concerned?

### DISCUSSION THREAD

**Discussion by dennisrahmen** [ant-design-blazor/ant-design-blazor#2020] [General]
I think the following video does a really good job at describing the current fuss about the .NET Foundation.
Also, there are all the important links in the video description:
https://www.youtube.com/watch?v=ui1Wg8oiHgc

Should we be concerned?

**Comment by ElderJames** [comment]
For the moment, the .NET Foundation has little impact on us. We are completely autonomous and the Foundation has not given us any guidance.

Do you think we need to do something about it?

**Reply by anddrzejb** [reply]
If the project is not under the Foundation, I doubt we should do anything. Let's wait out the storm and see what survives. Maybe they become somewhat more...helpful?

**Reply by dennisrahmen** [reply]
@anddrzejb  If that's the case then waiting would be the best option.

Was just concerned the Foundation and the current discussion could impact this project.
